src/components/external/DisputeManagement/components/DefendDisputeFlow/DefendDisputeFlow.scss

```
@@ -0,0 +1,74 @@
+@use '../../../../../style';
+
+.adyen-pe-defend-dispute {
+    &__container {
+        display: flex;
+        flex-direction: column;
+        gap: style.token(spacer-070);
+        padding-top: style.token(spacer-070);
+        width: 100%;
+    }
+
+    &-document-requirements {
+        font-family: system-ui;
+
+        &--list {
+            margin: 0;
+            padding-left: style.token(spacer-070);
+        }
+    }
+
+    &__title {
+        color: #000;
+        font-weight: 700;
+    }
+
+    &__error-icon {
+        color: style.token(color-background-critical-strong);
+        height: style.token(spacer-100);
+        width: style.token(spacer-100);
+
+        svg {
+            height: inherit;
+            width: inherit;
```

I think what you need here is a style declaration to get the desired icon size.
```suggestion
            display: block;
            font-size: 16px;
```